ansible/bootstrap.yml

```diff
@@ -226,25 +226,22 @@
     - selinux
     - update
   tasks:
-    - name: Check for pending reboot from package updates
-      stat: 
-        path: /var/run/reboot-required
-      register: update_reboot_required
-    - debug:
-        msg: "setstatus:{{ (sestatus.reboot_required | default(false)) }} packages: {{ (update_reboot_required.stat.exists | bool) }}"
-    - name: Reboot if required from SELinux state change or package upgrades
+    - name: Reboot to cover SELinux state change or package upgrades
       reboot:
         post_reboot_delay: 30
-      when: (sestatus['reboot_required'] | default(false)) or (update_reboot_required.stat.exists | bool)
     - name: Wait for hosts to be reachable
       wait_for_connection:
         sleep: 15
-    - name: update facts
```

Comment:
Do we still need to refresh the facts? Not sure if they contain stuff like running kernel version?

Reply:
I don't _think_ we need to, because either the ofed or the ansible_init tasks will gather facts anyway?

Reply:
I think the [new tasks ](https://github.com/stackhpc/ansible-slurm-appliance/pull/427/files#diff-7bc925a3ec3c73e00c9a56a23ae1ebaf83905d08b86cd39569091e1f89b32e42R240) only run if the host is in the ofed group. Should we run them against any host that has rebooted?

Reply:
Ah I see what you mean, yes that sounds sensible.

Reply:
Made the change